### PR TITLE
[Employer] update company

### PR DESCRIPTION
**Pull request by kominam**
------------------------------------------
[x] Build Fail!
------------------------------------------

[o] reek: success
[x] rubocop: failed
[o] rspec: success
[o] scss-lint: success
[o] brakeman: success
[o] eslint: success
[o] bundle-audit: success
[o] rails_best_practices: success
Rubocop của `education/management/permissions_controller.rb`

### CODE REVIEW

File: app/controllers/employer/base_controller.rb [hunk 1]
```diff
@@ -1,3 +1,10 @@
 class Employer::BaseController < ApplicationController
   layout "employer/layouts/application"
+  before_action :load_company
+
+  protected
+
+  def load_company
+    @company = Company.find_by id: params[:company_id]
```
Comment (phamvanbk11) [inline app/controllers/employer/base_controller.rb]: không tìm thấy thì sao

File: app/views/employer/companies/_edit_company_info.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="row">
   <div class="col-md-10 container">
     <div class="static-main-body">
-      <%= form_for :company do |f| %>
+      <%= form_for [:employer, @company], method: :put do |f| %>
```
Comment (phamvanbk11) [inline app/views/employer/companies/_edit_company_info.html.erb]: sao phải thêm method put

File: app/controllers/employer/base_controller.rb [hunk 3]
```diff
@@ -1,3 +1,10 @@
 class Employer::BaseController < ApplicationController
   layout "employer/layouts/application"
+  before_action :load_company
+
+  protected
+
+  def load_company
```
Comment (nhunght) [inline app/controllers/employer/base_controller.rb]: dùng load_resource :company

File: app/assets/javascripts/employer.js [hunk 4]
```diff
@@ -22,3 +22,4 @@
 //= require flash_message
 //= require common
 //= require employer/candidate
+//= require employer/image
```
Comment (nhunght) [inline app/assets/javascripts/employer.js]: require cả folder đi

**Pull request merged**